CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -511,14 +511,16 @@ public static function getPartialPaymentWithType($entityId, $entityName = 'parti
    */
   public static function getTotalPayments($contributionId) {
     $statusId = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Completed');
+    $refundStatusId = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Refunded');
 
     $sql = "SELECT SUM(ft.total_amount) FROM civicrm_financial_trxn ft
       INNER JOIN civicrm_entity_financial_trxn eft ON (eft.financial_trxn_id = ft.id AND eft.entity_table = 'civicrm_contribution')
-      WHERE eft.entity_id = %1 AND ft.is_payment = 1 AND ft.status_id = %2";
+      WHERE eft.entity_id = %1 AND ft.is_payment = 1 AND ft.status_id IN (%2, %3)";
```

Comment:
@monishdeb @eileenmcnaughton @pradpnayak seems calculating total payment from `financial_trxn` table should also consider the negative amount value with refund status? We do the same in [`getPartialPaymentWithType`](https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/FinancialTrxn.php#L481) function. Thoughts - if this seems fine to you?

Reply:
Apprently it doesn't seem odd to me consider Refunded status. But I checked the code for its impact and this function get called at two places:
```CRM//Core/BAO/FinancialTrxn.php:536:    $paid = CRM_Core_BAO_FinancialTrxn::getTotalPayments($params['contribution_id']);
CRM//Price/BAO/LineItem.php:1176:    $paidAmount = CRM_Core_BAO_FinancialTrxn::getTotalPayments($contributionId);
```
As per the usage I played with some Event's change-fee scenario  and line-item editor to check how it behaves but didn't found any unintended behaivour. So this seems fine to me. 

(Although it would be ok to get rid of the status subclause, but let's not tweak this rule in this PR and it might cause regression)